src/lib/feed.js

```diff
@@ -79,7 +61,7 @@ export default class StreamFeed {
      * @return {Promise} Promise object
      */
 
-    activity = replaceStreamObjects(activity);
+    activity = utils.replaceStreamObjects(activity);
     if (!activity.actor && this.client.currentUser) {
       activity.actor = this.client.currentUser.ref();
     }
```

Comment:
any chance not replacing the object interferes with this check?

Reply:
Not likely since `activity.actor` would be truthy if replaced. It's fine otherwise.